### ISSUE TITLE
File not found: __reskins-bobs__/..../rubber.png

### ISSUE THREAD

**Issue by Henriquelay** [kirazy/reskins-bobs#6]
First of all: Great work!
![image](https://user-images.githubusercontent.com/37563861/95935998-e905b980-0daa-11eb-9a78-2acc38e0c953.png)
I have tried several things, couldn't get this mod to work with my Bob's Electronics for some reason, they just really wanna fight.
There is only Bob's Electronics, Artisanal Reskins: Bob's Mods and it's dependencies active.

I can't find `rubber.png` on Bob's Electronics' `/graphics/icons/plates/items/` but rather on `/graphics/icons/`. ![`/graphics/icons/`](https://user-images.githubusercontent.com/37563861/95936502-1a32b980-0dac-11eb-892e-f85fbb953dde.png)

Whereas on `reskins-bobs` has said folder, but no signal of `rubber.png` there. ![no signal of `rubber.png` there](https://user-images.githubusercontent.com/37563861/95936608-63830900-0dac-11eb-8810-5df9b54c689e.png)

**Comment by Henriquelay** [comment]
I have imported the `rubber.png` from ShinyGFX and scaled it to fit 64x64 (required by your mod) and put into that folder, worked fine.
![image](https://user-images.githubusercontent.com/37563861/95937175-8d88fb00-0dad-11eb-8122-d3cb0d5bdb5b.png)
Rubber.png on yours is just missing for me?

**Comment by kirazy** [comment]
Woops, I forgot that rubber was referenced in two places and neglected to update the reference in the electronics items file to point to the new location. Commit 5b84c9475cc7dd7e402851d0318ad7eee14e9a0d fixes this. Will release an update shortly.

The rubber icon it's looking for was moved into reskins-library under graphics/icons/shared/items but I neglected to update the reference. I'm verifying the fix right now (Factorio loads slowly...)
